lab-taylor/gulpfile.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
This is a great start!  There are a few improvements you can make though, see my comments below for more information.